WowUp.WPF/Utilities/StartupOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CommandLine;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WowUp.Common.Enums;
+
+namespace WowUp.WPF.Utilities
+{
+    public class StartupOptions
+    {
+        [Option(shortName: 'i', longName: "install", HelpText = "Specify addon urls to install them")]
+        public IEnumerable<string> InputURLs { get; set; }
+        [Option(shortName: 's', longName: "silent", HelpText = "Start application minimized")]
```

Comment:
can we change this to 'm' and "minimized"

Reply:
Sure

Reply:
Actually, now when I though about it a bit, I am not so sure :)
Considering your advice about handling this argument in MainWindow, I imagined the silent mode as launching the app without showing the main window at all. The tray icon should appear but that is it. And starting it minimized is kind of a different thing. I do not really see a purpose in doing so. Can you explain?

Reply:
The system tray icon is tied to the main window. When the app is running in the background all that means is the main window display state is set to Hidden. its actually still there. 

See <tb:TaskbarIcon in MainWindow.xml

You could just add a check to the OnSourceInitialized in MainWindowViewModel to set the state to Hidden if that arg was passed in.

Reply:
Ok, I see. I could have sworn that it worked without showing the main window. But the latest build does not behave in this way anymore. Didn't find WindowState.Hidden though. Calling Hide() and setting ShowInTaskBar = false seems to make the required work.